render/detailed/summary.go

```diff
@@ -311,16 +293,38 @@ func hostNodeSummary(base NodeSummary, n report.Node) (NodeSummary, bool) {
 		base.Label = hostname
 	}
 
-	if h, ok := n.Counters.Lookup(report.Host); ok {
-		base.Stack = true
-		if h == 1 {
-			base.LabelMinor = fmt.Sprintf("%d host", h)
-		} else {
-			base.LabelMinor = fmt.Sprintf("%d hosts", h)
+	return base, true
+}
+
+// groupNodeSummary renders the summary for a group node. n.Topology is
+// expected to be of the form: group:container:hostname
+func groupNodeSummary(base NodeSummary, n report.Node) (NodeSummary, bool) {
+	parts := strings.Split(n.Topology, ":")
+	if len(parts) != 3 {
+		return NodeSummary{}, false
+	}
+
+	label, ok := n.Latest.Lookup(parts[2])
+	if !ok {
+		return NodeSummary{}, false
+	}
+	base.Label, base.Rank = label, label
+
+	if count, ok := n.Counters.Lookup(parts[1]); ok {
+		base.LabelMinor = fmt.Sprintf("%d %s", count, parts[1])
+		if count != 1 {
+			if strings.HasSuffix(parts[1], "s") {
+				base.LabelMinor += "es"
+			} else {
+				base.LabelMinor += "s"
+			}
```

Comment:
This is generally frowned upon (programatically modifying text) as it localises very poorly.

I'd go for a lookup table (or a humanLabel & humanLabelPlural on the topology?)